src/dialogxml/widgets/scrollbar.cpp

```diff
@@ -62,6 +62,10 @@ void cScrollbar::setStyle(eScrollStyle newStyle) {
 	style = newStyle;
 }
 
+void cScrollbar::set_mousewheel_event_catching_rect(rectangle rect) {
```

Comment:
This name is too long... can we call it something like `set_controlled_rect` or `set_pane_rect` (after the cScrollPane class)?

Reply:
done